images/casperjs/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM registry.fedoraproject.org/fedora:latest
+
+RUN yum -y update && yum install -y npm bzip2 fontconfig python3
```

Comment:
Please add `yum clean all` at the end of all these. It should reduce the size of the image even further

Reply:
YES!  Good grief, thank you for reminding me of that.

I'm adding `rm -rf /var/cache/yum/*` also per https://access.redhat.com/solutions/1989973.  That will squeeze out another ~100 MiB.

Reply:
$ buildah images
REPOSITORY                          TAG      IMAGE ID       CREATED          SIZE
localhost/yaspeller                 latest   0d785bb592e5   2 minutes ago    351 MB
localhost/userguide                 latest   c779272cefc0   10 minutes ago   332 MB
localhost/casperjs                  latest   3f7b28d6f00a   12 minutes ago   458 MB

Reply:
Saving the planet!